diagnostics-app/src/app/app.component.ts

```diff
@@ -8,32 +8,29 @@
  */
 
 import { Component, HostBinding, OnInit } from '@angular/core';
+import { Theme } from './core/enums/theme.enum';
+import { ThemeService } from './core/services/theme.service';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css'],
 })
 export class AppComponent implements OnInit {
-  private _theme: string = 'dark';
-  @HostBinding('class') private _cssClass!: string;
-
-  get theme() {
-    return this._theme;
-  }
+  @HostBinding('class') private _cssClass!: Theme;
```

Comment:
What is the added value from changing string to Theme?

Reply:
Same as below.